static/assets/js/feedback.js

```diff
@@ -80,13 +81,54 @@ const holdSendButton = seconds => {
     step()
 }
 
+const setPreSendWarning = (heading, description) => {
+    setStatus("", "warning")
+    const headingEl = document.createElement('p')
+    headingEl.textContent = heading
+    statusEl.appendChild(headingEl)
+    headingEl.insertAdjacentHTML('afterbegin', `<b>${i18n.preSendWarning.warning}</b> `)
+    const descriptionEl = document.createElement('p')
+    descriptionEl.textContent = description
+    statusEl.appendChild(descriptionEl)
+    const overrideEl = document.createElement('p')
+    overrideEl.textContent = i18n.preSendWarning.override
+    overrideEl.classList.add('mb-0')
+    statusEl.appendChild(overrideEl)
+    holdSendButton(5)
+}
+
+const preSendCheck = content => {
+    content = content.toLowerCase()
+    let warningText = false
+    const notSTWords = 'scratch team,st'.split(',')

```

Comment:
regex these

Reply:
and also use `\b`, what if i say the word "post" for whatever reason